playbooks/templates/.env.j2

```diff
@@ -197,6 +197,7 @@ ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access | string
 # REQUIRED when ACCOUNTS_LIMIT_ACCESS is not 'false': manually defined by user, or default will be used
 ACCOUNTS_TEST_USER_EMAIL={{ webportal_server_config.accounts_test_user_email }}
 ACCOUNTS_TEST_USER_PASSWORD={{ webportal_server_config.accounts_test_user_password }}
+ACCOUNTS_TEST_USER_API_KEY={{ webportal_server_config.accounts_test_user_api_key }}
```

Comment:
I think that docs comment doesn't match: Default is empty which I think is not a valid value (2 above vars email/password have generated valid defaults, no further user intervention is needed).

Reply:
Good point. I talked to Karol and updated the docs to match the real situation.